src/Microsoft.Extensions.Caching.Redis/RedisCache.cs

```diff
@@ -380,6 +426,14 @@ private void Refresh(string key, DateTimeOffset? absExpr, TimeSpan? sldExpr)
             return absoluteExpiration;
         }
 
+        private void LogSuppressedException(Exception ex)
+        {
+            if (_logger != null)
+            {
+                _logger.LogDebug("An exception during cache operation was suppressed: {message}", ex.ToString());
```

Comment:
you don't need to insert the message, you can pass down the entire exception object.